database/migrations/2019_12_25_0600_config_exiftool_ternary.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/** @noinspection PhpUndefinedClassInspection */
+use App\Configs;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+class ConfigExiftoolTernary extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		if (!defined('BOOL')) {
+			define('BOOL', '0|1');
+		}
+		if (!defined('TERNARY')) {
+			define('TERNARY', '0|1|2');
+		}
+
+		if (Schema::hasTable('configs')) {
+			Configs::where('key', '=', 'has_exiftool')
```

Comment:
I would also add something in the like:
```php
->whereNull('value')
```

but I think in the current state, this is also fine.

Reply:
One could ask why we don't do the 
```php
			exec('which exiftool 2>&1 > /dev/null', $output, $status);
```
check in the migration too. :)

Reply:
> I would also add something in the like:
> 
> ```
> ->whereNull('value')
> ```
> 
> but I think in the current state, this is also fine.

We're moving from bool to ternary -> imho, we should always overwrite to be on the safe side